<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="manoj" failures="0" tests="4" name="com.comcast.crm.contacttest.CreateContactTest" time="81.785" errors="1" timestamp="2024-08-08T14:10:05 IST" skipped="0">
  <testcase classname="com.comcast.crm.contacttest.CreateContactTest" name="CreateContactWithOrgTest" time="38.191"/>
  <testcase classname="com.comcast.crm.contacttest.CreateContactTest" name="createContactTest" time="20.262"/>
  <testcase classname="com.comcast.crm.contacttest.CreateContactTest" name="createContactWithSupportDateTest" time="23.332"/>
  <testcase classname="com.comcast.crm.contacttest.CreateContactTest" name="configAS" time="0.000">
    <error message="Cannot invoke &quot;java.sql.Connection.close()&quot; because &quot;this.con&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.sql.Connection.close()" because "this.con" is null
at GenericUtility.DatabaseUtility.closeDBconnection(DatabaseUtility.java:40)
at GenericUtility.BaseClass.configAS(BaseClass.java:78)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:410)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- configAS -->
</testsuite> <!-- com.comcast.crm.contacttest.CreateContactTest -->
